includes/solr.inc

```diff
@@ -131,8 +131,13 @@ function islandora_ocr_map_highlighted_fields_to_words(HOCR $hocr, array $search
   $highlighted_matches = array();
   foreach ($snippets as $snippet) {
     $matches = $hocr->search($snippet, $search_params);
+    $word_id = 1;
     foreach ($matches as $match) {
-      $id = $match['id'];
+      // Some hOCR implementations don't provide unique IDs for words. In this
+      // case let's grab a numerical one. It doesn't get used in any of the
+      // theme layer code that currently exists.
+      $id = isset($match['id']) ? $match['id'] : $word_id;
+      $word_id++;
```

Comment:
Is the expectation here that if there's an ID on one match, there's an ID on all of them? Or is there any chance we could have overlapping IDs using this method? Could be safer to namespace it? I dunno, I'm not 100% on the implementation